src/translations/nbTranslations.json

```diff
@@ -1497,6 +1497,11 @@
       "files_awaits_approval_other": "{{count}} filer venter på å bli godkjent",
       "files_completed": "Ferdig",
       "files_count": "Filer ({{count}})",
+      "files_helper_text": "<p>Her kan du laste opp fulltekst av publikasjonen og andre dokumenter som supplement og appendiks.</p>",
+      "files_helper_text_paragraph_2": "Du kan laste opp dokument(er) som spesifiserer kravene til offentliggjøring, som hvilken versjon du kan offentliggjøre, lisens og eventuell dato for utsatt tilgjengeliggjøring (sperrefrist/embargo). For eksempel er forfatteravtalen inngått med forlaget til hjelp når kurator skal sjekker om og når filen kan gjøres offentlig i NVA.",
```

Comment:
Her blir jeg også litt forvirret med at `files_helper_text` fikser `<p>` selv, mens `files_helper_text_paragraph_2` ikke støtter det, og oversetter må egentlig manøvrere seg mellom alle disse og rekkefølgene mellom dem for å justere på innhold